README.md

```diff
@@ -4,11 +4,7 @@
 
 This repository contains PowerShell cmdlets for developers and administrators to develop, deploy, and manage Microsoft Azure applications.
 
-Try it out in Azure Cloud Shell!
-* Direct link: Open a browser to https://shell.azure.com
```

Comment:
As https://shell.azure.com now redirects to https://portal.azure.com/#cloudshell, it is unnecessary to show them as 2 items anymore.